src/iexamine.cpp

```diff
@@ -1608,7 +1608,7 @@ std::list<item> iexamine::get_harvest_items( const itype &type, const int plant_
     const auto add = [&]( const itype_id &id, const int count ) {
         item new_item( id, calendar::turn );
         if( new_item.count_by_charges() && count > 0 ) {
-            new_item.charges = count;
+            new_item.charges *= count;
```

Comment:
Is there a way to get the default number of charges from the definition to make this part more clear?